Kitodo/src/main/webapp/WEB-INF/templates/includes/processes/executeScriptSelectedPopup.xhtml

```diff
@@ -34,7 +34,7 @@
                             <div>
                                 <p:commandButton value="addRole"
                                                  update="executeScriptSelectedForm:selectionScriptFieldTextArea"
-                                                 onclick="document.getElementById('executeScriptSelectedForm:selectionScriptFieldTextArea').value='action:addRole &quot;steptitle:TITLE_STEP&quot; group:GROUP_NAME'"/>
+                                                 onclick="document.getElementById('executeScriptSelectedForm:selectionScriptFieldTextArea').value='action:addRole &quot;steptitle:TITLE_STEP&quot; role:ROLE_NAME'"/>
```

Comment:
```suggestion
                                                 onclick="document.getElementById('executeScriptSelectedForm:selectionScriptFieldTextArea').value='action:addRole steptitle:TITLE_STEP role:ROLE_NAME'"/>
```

Reply:
I've testet it. it works without the quotation marks, they are just confusing
I would also suggest to change `steptitle` to `tasktitle`
You need to adjust this here and in KitodoScriptService.java l. 61